ts/.eslintrc

```diff
@@ -37,6 +37,7 @@
         "no-undefined": 0,
         "no-underscore-dangle": 0,
         "no-unused-expressions": 0,
+        "no-use-before-define": 0,
```

Comment:
@bre1470 Do we need to disable this? I am rather in favor of keeping this rule and instead move the isNumber to before it is used.

Reply:
The rule seems to be quite buggy, because it does not realize, that the function is already available at that line of the code where it complains. I see no point in having faulty rules turned on.

Reply:
And if we have a situation where it is really a problem, then our unit tests will right away fail.

Reply:
Actually TypeScript would even before that complain about it and not transpile the code at all.

Reply:
Hmm.. maybe `"no-use-before-define": ["error", { "functions": false, "classes": true, "variables": true }]`?

Reply:
We decided to disable the rule for now